llvm/test/tools/llvm-objcopy/ELF/respect-umask.test

```diff
@@ -4,7 +4,6 @@
 ## Windows has no umask so this test makes no sense, nor would
 ## it work because there is no umask(1) in a Windows environment
```

Comment:
```suggestion
## Windows has no umask so this test makes no sense.
```

Reply:
Seems fine to me. I didn't write the original comment. Perhaps it was trying to explain that Windows has neither umask(1) nor umask(2) ?!

Reply:
Probably it was. But that's not relevant anymore now that the internal shell has umask.